docs/SETUP_GUIDE.md

```diff
@@ -4,48 +4,108 @@ Here are the steps required to setup an instance of the rolling demo on your own
-- Populate all the variables mentioned in the file with your secrets.
+```bash
+# Github secrets
+# For more information on how to setup your github app
+# you can take a look here:
+# https://github.com/redhat-ai-dev/ai-rhdh-installer/blob/main/docs/APP-SETUP.md
+export GITHUB_APP_APP_ID="your-github-apps-app-id"
+export GITOPS_GIT_TOKEN="your-github-token"
+export GITOPS_GIT_ORG="your-github-org-name"
+export GITHUB_APP_CLIENT_ID="your-github-app-client-id"
+export GITHUB_APP_CLIENT_SECRET="your-github-app-client-secret"
+export GITHUB_APP_WEBHOOK_URL="your-github-app-webhook-url"
+export GITHUB_APP_WEBHOOK_SECRET="your-github-app-webhook-secret"
+export GITHUB_APP_PRIVATE_KEY="your-github-org-name"
+
+# ArgoCD secrets
+export ARGOCD_USER="your-user" # default value is "admin"
+
+# RHDH secrets
+export BACKEND_SECRET="a-randomly-generated-string"
+# RHDH_CLUSTER_ROUTER_BASE: should contain the domain suffix of all the
+# OCP routes.
+export RHDH_CLUSTER_ROUTER_BASE="apps.mycluster.openshift.com"
+export RHDH_BASE_URL="https://rolling-demo-backstage-rolling-demo-ns.${RHDH_CLUSTER_ROUTER_BASE}" # this is the route for the RHDH console
+export RHDH_CALLBACK_URL="${RHDH_BASE_URL}/api/auth/oidc/handler/frame"
+
+# RHDH Postgres Secrets - any valid password strings for these two env vars will do
+export POSTGRESQL_POSTGRES_PASSWORD="your-preffered-postgres-pass"
+export POSTGRESQL_USER_PASSWORD="your-preffered-user-pass"
+
+# Quay.io secrets
+# For more information on how to setup quay.io you check the doc
+# here: https://github.com/redhat-ai-dev/ai-rhdh-installer/blob/main/docs/APP-SETUP.md#quay-setup
+export QUAY_DOCKERCONFIGJSON="your-quay.io-dockerconfig.json"
+
+# KeyCloak (RH SSO) secrets
+# In this area you can point to an already existing keycloak instance.
+# As per RHDHPAI use case please refer to the already existing secrets in our Dev Instance.
+export KEYCLOAK_METADATA_URL="{your-instance's-url}/auth/realms/your-realm"
+export KEYCLOAK_CLIENT_ID="your-client-id"
+export KEYCLOAK_CLIENT_SECRET="your-secret"
+# KEYCLOAK_REALM: The realm you want to use for your deployment.
+# Check more info for realms here: https://www.keycloak.org/docs/latest/server_admin/index.html#_configuring-realms
+export KEYCLOAK_REALM="your-realm"
+export KEYCLOAK_LOGIN_REALM=${KEYCLOAK_REALM}
+export KEYCLOAK_BASE_URL="{your-instance's-url}/auth"
+
+# NOTE: For Ollama & Lightspeed we rely on 3Scale. Therefore,
+# to setup your Ollama & Lightspeed tokens & urls you'll need
+# first to register an application on the 3Scale service.
+# Check more info here: https://docs.redhat.com/en/documentation/red_hat_3scale_api_management/2.11/html/getting_started/first-steps-with-threescale_configuring-your-api
+# As per RHDHPAI use case please refer to the already existing secrets in our Dev Instance.
+
+# Ollama secrets
+export OLLAMA_TOKEN="your-ollama-token"
+export OLLAMA_URL="ollama-url"
```

Comment:
let's just put the dev cluster's ollama url and then a comment above that you can point to another one if another one is defined